fideslog/sdk/python/client.py

```diff
@@ -91,17 +73,42 @@ def register(self, registration: Registration) -> None:
         Register a new user.
         """
 
-        __set_event_loop()
+        self.__set_event_loop()
         run(self.__send(registration))
 
     def send(self, event: AnalyticsEvent) -> None:
         """
         Record a new analytics event.
         """
 
-        __set_event_loop()
+        self.__set_event_loop()
```

Comment:
This does seem pretty unnecessary to include - if we're doing this during `__init__(...)` then this'll always just be a no-op.

As a defensive inclusion though, I can see why it might be needed. There is code that calls the protected `_AnalyticsClient__send` directly, which I don't quite understand why: https://github.com/ethyca/fides/blob/5a485387d8af247ec6479e4115088cbbb8394d77/src/fides/api/ops/analytics.py#L44

Reply:
In other words, LGTM- but still smells funny.